### PR TITLE
ac-34 money transactions api doc

### PR DESCRIPTION
**Pull request by ExtraGravity**

### CODE REVIEW

File: src/routers/transactions/money/index.ts [hunk 1]
```diff
@@ -11,16 +11,58 @@ import productPayments from '@/routers/transactions/money/productPayments';
 
 const router = createRouter();
 
+/**
+ * @api {get} /transactions/money
+ * @description Returns all money transactions.
+ *              Parameters have no effect on this request.
+ * @apiName GetAllMoneyTransactions
+ *
+ * @apiError (401) Unauthorized - Must be admin
```
Comment (beaesguerra) [inline src/routers/transactions/money/index.ts:20]: 403

File: src/routers/transactions/money/index.ts [hunk 2]
```diff
@@ -11,16 +11,58 @@ import productPayments from '@/routers/transactions/money/productPayments';
...
+ *
+ * @apiSuccess (200) {String} Successfully retrieved all money transactions
```
Comment (beaesguerra) [inline src/routers/transactions/money/index.ts:22]: need the "Success" word first so it doesnt get formatted weird again?

File: src/routers/transactions/money/index.ts [hunk 3]
```diff
@@ -11,16 +11,58 @@ import productPayments from '@/routers/transactions/money/productPayments';
...
+ * @apiSuccessExample Success-Response:
+  [
+    {
+      "moneyTransactionUuid": "43c542e4-443e-45fa-91d5-f073161eb216",
+      "datetime": "2018-01-23 04:05:06",
```
Comment (beaesguerra) [inline src/routers/transactions/money/index.ts:27]: see my question in other PR about UTC format

File: src/routers/transactions/money/index.ts [hunk 4]
```diff
@@ -11,16 +11,58 @@ import productPayments from '@/routers/transactions/money/productPayments';
...
+      "toPersonUuid": "3cad5e7c-5444-4de1-aa81-a7d15acb35f1",
+      "fromPersonUuid": "43c542e4-443e-45fa-91d5-f073161eb216",
+      "amount": 223.42,
```
Comment (beaesguerra) [inline src/routers/transactions/money/index.ts:30]: just out of curiosity, in this example, it has a precision and in your other pull request there was a precision of 4. Are we going to be making sure that the precision is consistent? or does it not matter too much?

File: src/routers/transactions/money/index.ts [hunk 5]
```diff
@@ -11,16 +11,58 @@ import productPayments from '@/routers/transactions/money/productPayments';
...
 
-router.get('/:uuid', async (req, res) => {
-  res.status(StatusCode.OK).send('Successfully retrieved money transaction');
-}, authorized(UserType.ADMIN));
-
 router.use('/loans', loans)
 router.use('/loanPayments', loanPayments)
 router.use('/productPayments', productPayments)
 
+// dynamic api needs be after the static ones
```
Comment (beaesguerra) [inline src/routers/transactions/money/index.ts:43]: is this just a style thing?

File: src/routers/transactions/money/index.ts [hunk 6]
```diff
@@ -11,16 +11,58 @@ import productPayments from '@/routers/transactions/money/productPayments';
...
+/**
+ * @api {get} /transactions/money/<uuid>
+ * @description Returns a specific money transaction.
+ *              Parameters have no effect on this request.
+ * @apiName GetMoneyTransaction
+ *
+ * @apiError (401) Unauthorized - Must be admin
```
Comment (beaesguerra) [inline src/routers/transactions/money/index.ts:50]: forbidden

**Review by beaesguerra — CHANGES_REQUESTED** [review]

**Comment by beaesguerra** [comment]
oh! we don't need to do money transactions yet?

**Comment by ExtraGravity** [comment]
Yea, I'll fix this up when we need it.

**Pull request closed without merging**